padertorch/train/hooks.py

```diff
@@ -590,8 +590,10 @@ def set_best_symlink(self, ckpt_dir):
         best_ckpt_path = ckpt_dir / self._best_ckpt_name
         if best_ckpt_path.is_symlink():
             best_ckpt_path.unlink()
-        best_ckpt_path.symlink_to(self.ckpt_ranking[0][0])
-
+        try:
+            best_ckpt_path.symlink_to(self.ckpt_ranking[0][0])
+        except FileExistsError:
+            raise FileExistsError('Best checkpoint needs to be a symlink to a checkpoint, not a file!')
```

Comment:
Can you make two changes?
 - Add symlink name and target to the exeption msg.
 - You could do `raise FileExistsError(...) from None`. That would hide the previous exception.
    - Alternatively use `raise FileExistsError(...) from e`

Reply:
I think the target name is sufficient here because the error comes from the target path being a file instead of a symlink.